tools/integration_tests/read_cache/cache_file_for_range_read_false_test.go

```diff
@@ -148,8 +148,8 @@ func TestCacheFileForRangeReadFalseTest(t *testing.T) {
 
 	// Run tests with parallel downloads disabled.
 	flagsSet := [][]string{
-		{"--implicit-dirs", "--config-file=" + createConfigFile(cacheCapacityForRangeReadTestInMiB, false, configFileName, false, getDefaultCacheDirPathForTests())},
-		{"--config-file=" + createConfigFile(cacheCapacityForRangeReadTestInMiB, false, configFileName, false, ramCacheDir)},
+		{"--implicit-dirs", "--config-file=" + createConfigFile(cacheCapacityForRangeReadTestInMiB, false, configFileName, false, getDefaultCacheDirPathForTests(), false)},
+		{"--config-file=" + createConfigFile(cacheCapacityForRangeReadTestInMiB, false, configFileName, false, ramCacheDir, false)},
```

Comment:
@charith87 I am going to revert the changes under integration_tests directory because right now there is a bug in how we create config-file and use it in tests. I will fix that in a separate PR and enable these tests post that.

Reply:
Sg. does that mean the previous tests were not  running correctly

Reply:
Yes, the previous tests were not run properly (not just for o-direct but maybe some other)

Reply:
Okk. Sg.